### PR TITLE
Add pyenv and virtual environment setup instructions

### PR DESCRIPTION
**Pull request by maishathasin**
## What

This PR addresses different Python versioning issues and mentions all the necessary setup steps to ensure a consistent development environment across different platforms .

## How
 It also includes instructions for setting up `pyenv`, installing the appropriate Python version, creating and activating virtual environments, this is updated in the **README.md**

## Why

To ensure that developers can easily set up their environment regardless of the operating system they are using, and to avoid conflicts and inconsistencies that arise from different Python versions.

### CODE REVIEW

**Comment by aishjayashankar** [comment]
A quick comment. The changes in this PR seems to be removing a few changes to the latest Readme, especially in the Config and Logging sections under "Running the code". Could you please check the latest changes and add the pyenv details on top of that? Thanks!

File: src/README.md [hunk 1]
```diff
@@ -1,28 +1,98 @@
...
+ #### On Windows:
 
-The tensorboard logs can be viewed by running tensorboard as follows:<br>
-`tensorboard --logdir=expt_dump/ --host 0.0.0.0`<br>
-Assuming `expt_dump` is the folder where the experiment logs are stored.
+```bash 
+git clone https://github.com/pyenv-win/pyenv-win.git %USERPROFILE%\.pyenv
+```
 
-After a successful run with 50 epochs, the Tensorboard experiment log should look something like below:
```
Comment (aishjayashankar) [inline src/README.md:26]: The changes in this readme removes these lines, plus the sample plot. Kindly recheck and update the same, thanks!

**Comment by tremblerz** [comment]
Hi @maishathasin Thanks for the PR. Can you please update your PR so the old guide is not removed

File: src/README.md (hunk 1 above)
Comment (maishathasin) [reply src/README.md]: Resolved!

**Comment by maishathasin** [comment]
> Hi @maishathasin Thanks for the PR. Can you please update your PR so the old guide is not removed

Resolved! , the new commit should not change the old Readme

**Comment by maishathasin** [comment]
@tremblerz  pinging again!

**Comment by tremblerz** [comment]
Hi @maishathasin sorry i didn't get notification before. Can you please confirm once again it is working fine?

**Comment by maishathasin** [comment]
hi @tremblerz Yes, this works just fine, no changes needed!

**Pull request merged**